WORKSPACE

```diff
@@ -464,12 +464,12 @@ HASKELL_LSP_COMMIT = "d73e2ccb518724e6766833ee3d7e73289cbe0018"
 
 HASKELL_LSP_HASH = "36b92431039e6289eb709b8872f5010a57d4a45e637e1c1c945bdb3128586081"
 
-GHC_LIB_VERSION = "8.8.0.20190704"
+GHC_LIB_VERSION = "0.20190715"
```

Comment:
Before we land this, we should get a stable version of our patched `ghc-lib` and refer to that one.

Reply:
Agreed.